src/popup.cpp

```diff
@@ -289,8 +296,11 @@ query_popup::result query_popup::query_once()
     }
     if( anykey ) {
         ctxt.register_action( "ANY_INPUT" );
-        // Mouse movement, button, and wheel
-        ctxt.register_action( "COORDINATE" );
```

Comment:
And I'm not quite sure about allowing mouse input by default because some old code might not expect it, especially when the popup is not set up to ask the user to pick an option.

Reply:
Cheers, it didn't occur to me that the mouse input was for keybindings. I added that back in and it's working as before:

![2022-06-04 15-44-59](https://user-images.githubusercontent.com/12537966/172023448-55eff7a5-aa0f-467b-a819-4daa74b920fa.gif)

I also switched mouse control to disabled by default for `query_popup`. `uilist` already handles "SELECT" so I haven't changed it.